### PR TITLE
Make pull request closed action more robust

### PR DESCRIPTION
**Pull request by floatplane**
Only try to delete the channel if it doesn't exist

### CODE REVIEW

**Comment by floatplane** [comment]
guess we'll find out if i did this right

File: .github/actions/expo/channel/delete/action.yaml [hunk 1]
```diff
@@ -23,5 +23,8 @@ runs:
       id: eas-update
       shell: bash
       run: |
-        eas channel:delete --non-interactive ${{ inputs.channel }}
-        eas branch:delete --non-interactive ${{ inputs.channel }}
+        eas channel:view ${{ inputs.channel }}  --non-interactive --limit 1
+        if [ $? -eq 0 ]; then
```
Comment (stevekuznetsov) [inline .github/actions/expo/channel/delete/action.yaml:27]: you can also

```sh
if eas channel:view ${{ inputs.channel }}  --non-interactive --limit 1; then
```

Comment (stevekuznetsov) [reply .github/actions/expo/channel/delete/action.yaml:27]: Which you'd need to do if you had `set -o errexit` set

Comment (floatplane) [reply .github/actions/expo/channel/delete/action.yaml:27]: it seems like that may be implicitly set by the GH job runner, since my approach didn't work!

Comment (floatplane) [reply .github/actions/expo/channel/delete/action.yaml:27]: thanks for the tip :)

**Pull request merged**